test/e2e/config/ironic.yaml

```diff
@@ -33,7 +33,7 @@ intervals:
   inspection/wait-registration-error: ["1m", "5s"]
   external-inspection/wait-available: ["20s", "1s"]
   default/wait-inspecting: ["2m", "10s"]
-  default/wait-available: ["10m", "10s"]
+  default/wait-available: ["10m", "1s"]
```

Comment:
I've not checked, but I wonder if this breaks anything? Is it enough for vbmc, and do we use this wait for anything else?

Reply:
Do you mean the change from `10s` to `1s`? I had some problems here while debugging and thought that the 10s interval check would have been able to have missed the state. That's why I changed it. But after finding the bug I don't think this was the problem.

Maybe I'm missing something here, what do you mean by breaks anything?

Reply:
The `"1s"` is just how often it checks. The deadline is `"10m"` still so this should not really change anything :slightly_smiling_face:

Reply:
Yes. Basically I was wondering if this wait time is used somewhere else, and if it is, would 1s be sufficient in those cases.

Reply:
Ah. Alright, I forgot about that xD Nevermind then.

Reply:
It is used in the Interval-test and in the Provisioning-test at least. But like lentzi90 is saying its just how often it checks. The limit before ending the check is till 10 minutes.